### ISSUE TITLE
The event submission form could be nicer!

### ISSUE THREAD

**Issue by isaacchansky** [codeforboston/calendar_api#1]
A few things right off the bat:
- no military time in date picker (No one I've come across has events listed in military time)
- The description option should explain how it's being used, maybe have a markdown-style syntax so you can add links
- maybe there should be a checkbox for free, as of now, in price idk if I should write "$X", "X", or if "free" is allowed 
-

**Comment by thecristen** [comment]
Is there a way to add events that don't have a specific time set ("August 2016" or "Week of June 4 2017")? I think it would be nice to have.. ;)

**Comment by isaacchansky** [comment]
I agree! There's currently no way to do that, but certainly would enable some different formats of "event" (i.e "Open data week").
